internal/server/server.go

```diff
@@ -97,6 +97,7 @@ func EchoEngine(ctrl Controller) *echo.Echo {
 	restricted.POST("/auth/sign_out", auth.Logout)
 	restricted.POST("/auth/update", auth.Update)
 	restricted.POST("/auth/change_pw", auth.UpdatePassword)
+	v1restricted.PUT("/users/:id/attributes/credentials", auth.UpdatePassword)
```

Comment:
`:id` is ignored there? We only use the connected user via its bearer token?

Reply:
Yes it was ignored. It think it should always be the same like in bearer token